SpanishFilter/sections/css_extended.txt

```diff
@@ -5,6 +5,7 @@
 yahoo.com#?#ul[class^="Mb(0)"] > li.js-stream-content:has(> div > a[rel="nofollow noopener noreferrer"]+div:contains(Patrocinado))
 !
 cokitos.com#?#.theiaStickySidebar > div.widget_text:has(> div.textwidget ins.adsbygoogle)
+elhacker.net#?#body > table:has(> tbody > tr > td > div.publi_sponsor)
```

Comment:
Is it required for main domain? I noticed this element only on `foro.elhacker.net`